src/cljs/vanilla/core.cljs

```diff
@@ -4,13 +4,34 @@
             [re-frame.core :as rf]
             [vanilla.widgets.simple-text]
             [vanilla.widgets.chart]
+            [vanilla.widgets.area-chart]
             [vanilla.widgets.bar-chart]
             [vanilla.widgets.dual-chart]
             [vanilla.widgets.pie-chart]
             [vanilla.widgets.side-by-side-chart]))
 
 
 (def widgets [
+              {:type        :area-chart
```

Comment:
next sprint let's add a new data source so we get some visual distinction and better show the area chart